system-test/datastore.ts

```diff
@@ -57,11 +57,11 @@ describe('Datastore', () => {
       publishedAt: new Date(),
       author: 'Silvano',
       isDraft: false,
-      wordCount: 400,
-      rating: 5.0,
+      wordCount: datastore.int(400),
+      rating: datastore.int(5.0),
```

Comment:
this feels a bit weird, because `5.0` looks like a floating point. How do we handle floating point values in this new world?